### DISCUSSION TITLE
Tool identification problem

### DISCUSSION THREAD

**Discussion by Duke1410** [projectdiscovery/httpx#723] [Q&A] [answered]
I can't recognize it at all. I extracted a lot of URLs. It is clear that the URL status is normal online, but the tools can't recognize it. There are too few.

**Comment by ehsandeep** [comment]
@Duke1410 do you have an example to share?

**Reply by prince-cheng** [reply]
echo http://youth.jzxy.edu.cn/ | ./httpx -sc -td -cl -method -title -silent -probe

**Reply by forgedhallpass** [reply]
@prince-cheng That domain cannot be resolved, that is why you get the [FAILED] message. That is expected behavior.

**Comment by forgedhallpass** [comment, marked as answer]
> @prince-cheng That domain cannot be resolved, that is why you get the [FAILED] message. That is expected behavior.
